Cargo.toml

```diff
@@ -33,6 +33,7 @@ rustversion = "1.0.15"
 
 # core
 num = "0.4.2"
+vtkio = { git = "https://github.com/elrnv/vtkio.git" }
```

Comment:
Any reason to not target a released version?

Reply:
While their README indicates that the latest release is `0.7.0`, the latest found on crates.io is `0.6.3`. I didn't want to check how much changes there are between versions, so I just used the latest commit.